app/app.py

```diff
@@ -1,6 +1,15 @@
+"""
+the main application
+"""
+
+from pathlib import Path
+from pprint import pprint
 from databasemanager import DatabaseManager
 
 if __name__ == "__main__":
     db_helper = DatabaseManager()
-    db_helper.get_all_data()
-    db_helper.close_connection()
\ No newline at end of file
+    db_helper.import_dataset(Path(__file__).parent /
+                             'datasets/prevalence-of-undernourishment.csv')
```

Comment:
use constant to store.

Reply:
should we put all constants into one file?

Reply:
i agree, at least for app level stuff like the dataset source